scenes/game/emote_display.gd

```diff
@@ -0,0 +1,36 @@
+extends Node2D
+
+@onready var panel : PanelContainer = $Panel
+@onready var image : TextureRect = $Panel/Margin/EmoteIcon
+@onready var text : Label = $Panel/Margin/EmoteText
+@onready var animation_player : AnimationPlayer = $AnimationPlayer
+
+func remove():
+	animation_player.stop()
```

Comment:
just to make sure, this is a different instance of an AnimationPlayer than the one that controls character animations right?

Reply:
yes